house/models.py

```diff
@@ -46,7 +46,7 @@ class Job(models.TextChoices):
 
 
 class House(models.Model):
-    house_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    user = models.OneToOneField(User, on_delete=models.CASCADE, primary_key=True)
```

Comment:
Do you have a way for a user to remove their account? (if no, maybe create an issue for it)

Reply:
I have added an issue for it 👍 #116